### PR TITLE
feat: added url box to access and discount coupons

### LINKED ISSUE #6472: Discount and Access Code: Make URL into a URL box and change the "actions icons" from vertical to horizontal
To improve the access and discount code overview please change the following:
1. Make the URL into a URL box like on the video area 
2. Change the "actions icons" from vertical to horizontal.

Screenshot from 2021-01-23 23-52-13

### PR DESCRIPTION
**Pull request by Creatoon**
Fixes #6472 

#### Short description of what this resolves:
This pr adds url box over access and discount coupons and and made cell-action buttons horizontal.

For Access code

Screenshot_2021-01-27 Access codes Tickets mailsac Events Open Event

For discount code

Screenshot_2021-01-27 Discount codes Tickets mailsac Events Open Event

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Creatoon** [comment]
@iamareebjamal  sir please review , thanks

**Review by maze-runnar — DISMISSED** [review]

File: app/templates/components/ui-table/cell/events/view/tickets/access-codes/cell-url.hbs [hunk 1]
```diff
@@ -1 +1 @@
-<a href='{{this.record}}' target='_blank' rel='noopener'> {{this.record}} </a>
+<UrlBox @url={{this.record}} />
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/events/view/tickets/access-codes/cell-url.hbs:1]:
```suggestion
<UrlBox @url={{this.record}} />

```

File: app/templates/components/ui-table/cell/events/view/tickets/discount-codes/cell-url.hbs [hunk 2]
```diff
@@ -1 +1 @@
-<a href='{{this.record}}' target='_blank' rel='noopener'> {{this.record}} </a>
+<UrlBox @url={{this.record}} />
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/events/view/tickets/discount-codes/cell-url.hbs:1]:
```suggestion
<UrlBox @url={{this.record}} />

```

**Comment by Creatoon** [comment]
@iamareebjamal sorry sir, i forgot to do that, and didn't seen your suggested change, i will take care of these mistakes from next times.

**Pull request merged**